roles/freshrss/tasks/main.yml

```diff
@@ -5,14 +5,34 @@
     state: directory
     # mode: 0755
   with_items:
+    - "{{ freshrss_data_directory }}/pgsql_data"
     - "{{ freshrss_data_directory }}/data"
     - "{{ freshrss_extensions_directory }}/extensions"
 
+- name: FreshRSS PostgreSQL Docker Container
+  docker_container:
+    name: freshrss-postgres
+    image: postgres:12-alpine
+    pull: true
+    volumes:
+      - "{{ freshrss_data_directory }}/pgsql_data:/var/lib/postgresql/data"
+    hostname: "freshrss-postgres"
+    ports:
+      - "{{ freshrss_db_port }}:5432"
```

Comment:
This isn't required in addition to the link in the freshrss definition

Reply:
Sorry but I'm not sure what you are talking about - line #526

Reply:
If you were talking about the hostname declaration I removed it. Something I have in my files and missed removing for your house style. 
If not - please expound as I'm lost.